### ISSUE TITLE
Discrepancies with roles available in Microsoft 365 admin center

### ISSUE THREAD

**Issue by Stephan-vNext** [MicrosoftDocs/OfficeDocs-O365ITPro#680]
Contrary to what is said on this page, some roles can't be found in the admin center, should it be with the new experience(https://admin.microsoft.com/AdminPortal/Home#/roles) or the old one (https://admin.microsoft.com/AdminPortal/Home#/users):
- Password Admin
- Security Admin

There's also a new role missing:
- Power BI Administrator

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1b4e45a4-3d00-c650-c40f-3a10b457cf46
* Version Independent ID: fefa373f-f380-8988-e58a-73e9255134bb
* Content: [About admin roles in the Microsoft 365 admin center](https://docs.microsoft.com/en-us/office365/admin/add-users/about-admin-roles?view=o365-worldwide#feedback)
* Content Source: [Office365-Admin/add-users/about-admin-roles.md](https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/blob/public/Office365-Admin/add-users/about-admin-roles.md)
* Service: **o365-administration**
* GitHub Login: @adefowler
* Microsoft Alias: **anfowler**

**Comment by mypil** [comment]
@officedocsbot assign @mypil

**Comment by mypil** [comment]
@Stephan-vNext  - Thank you for submitting feedback.

I will get this issue over to the O365ITPro writing team for investigation.

Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.

**Comment by adefowler** [comment]
@Stephan-vNext - I'm investigating and will get back to you with an answer.

**Comment by adefowler** [comment]
@Stephan-vNext: I found out what happened.

**PowerBI admin role** is in the admin center. You might not be able to find it because by default we first show a subset of the roles, those that are most commonly used by organizations. If you scroll down the bottom of the page and select "Show all roles", the Power BI role should magically appear.

As for the other two roles **Password admin** and **Security admin**, you accidentally got a preview of roles we haven't released yet. 

Per your comment, I've updated the article to add an FAQ question about why the role might be missing and removed the 2 roles that I shouldn't have written about. :-) 

Thank you so much for taking the time to let me know! It's greatly appreciated.
